### PR TITLE
Handle errors

### PR DESCRIPTION
**Pull request by rgould**

### CODE REVIEW

File: lib/image_upload.js [hunk 1]
```diff
@@ -182,6 +182,24 @@ ImageUpload = (function ImageUploadClosure() {
...
+      /*
+       * TODO: Better error handling than just console logging
+       */
+      console.error(error.message);
+      fileObj.on('error', function(store) {
+        /*
+         * fileObj.emit doesn't pass useful information here, so do nothing.
+         * Errors should have been handled by the store's error handler.
+         * https://github.com/CollectionFS/Meteor-cfs-collection/blob/64a0fc6e6a95468c5ff8dc9e81a9c56ecf9aeb6a/common.js#L94
+         */
```
Comment (pauldowman) [inline lib/image_upload.js:195]: We should at least log it though I think.

Comment (rgould) [inline lib/image_upload.js:195]: Agreed

Comment (rgould) [inline lib/image_upload.js:195]: @pauldowman I misread. We do log the error on line 189. Within the error handler starting on 190, there is no useful information to be logged. The callback isn't passed any error, just the same store that is passed to the callback in 185. I think it's fine to let the `fileObj` handler do nothing.

Comment (pauldowman) [inline lib/image_upload.js:195]: :ok:

**Pull request merged**